### ISSUE TITLE
After upgrade to nextjs CSS Import from node_modules not working properly

### ISSUE THREAD

**Issue by harshzalavadiya** [vercel/next.js#25422]
### What version of Next.js are you using?

10.2.2

### What version of Node.js are you using?

14

### What browser are you using?

chrome

### What operating system are you using?

linux

### How are you deploying your application?

vercel

### Describe the Bug

After upgrading nextjs to latest build works as expected but imported css from node_modules package isn't rendering properly when application is compiled in production mode, the issue was introduced at `next@10.2.1-canary.3`

### [next@latest](https://css-import-bug.vercel.app)
![image](https://user-images.githubusercontent.com/5774849/119385675-1bce4380-bce4-11eb-83fc-1a408ca2d54a.png)

### [next@downgraded](https://css-import-bug-git-downgraded-harshzalavadiya.vercel.app/)
![image](https://user-images.githubusercontent.com/5774849/119385745-36a0b800-bce4-11eb-9847-3b947cc4703c.png)

### Expected Behavior

It should render component same as earlier version used to do

### To Reproduce

https://github.com/harshzalavadiya/canary-bug

- Bug only occures when running on production build of `next@10.2.1-canary.3` and after versions
- NOT reproducable on development mode only after build is done

There are two branches
- main: contains `next@latest` version w/ simple quill-js component [vercel@master](https://css-import-bug.vercel.app/)
- downgraded: same code as master but running `next@10.2.1-canary.2` which works as expected [vercel@downgraded](https://css-import-bug-git-downgraded-harshzalavadiya.vercel.app/)

**Comment by harshzalavadiya** [comment]
Issue seems to be coming from 

https://github.com/vercel/next.js/blob/3b9221ff10f1d14672596e5df4407bf8dc4a7cd7/packages/next/build/webpack-config.ts#L1689-L1699

if I comment else part it starts importing CSS files from node_modules as expected (TBH I have no idea how/why it's happening this is just result of traceback to root issue)

**Comment by rafde** [comment]
This issue isn't limited to `*.css` files in `node_modules`, it also happens with any file in `node_modules` folder that webpack has to watch in dev-server mode

**Comment by JanKaifer** [comment]
Thes reproduction returns 404 it might be private.
This issue is quite old, is it still relevant in canary?
It might have been already fixed.

**Comment by balazsorban44** [comment]
This issue has been automatically closed because it wasn't verified against next@canary. If you think it was closed by accident, please leave a comment. If you are running into a similar issue, please open a new issue with a reproduction. Thank you.
